.github/workflows/example-ktx.yml

```diff
@@ -0,0 +1,35 @@
+name: Android CI
+
+on:
+  push:
+    branches: [ main ]
+    paths: [ '.github/**', 'examples/example-clientlib-ktx-app/**' ]
```

Comment:
Since this depends on the `clientlib` and `clientlib-ktx` sources, the `paths` filter for both `push` and `pull_request` should include `android/**` as well.